app/routes.py

```diff
@@ -1 +1,205 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, jsonify, abort, make_response, request
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from datetime import date
+import requests
+import os
+
+
+def validate_model(cls, model_id):
+    model = cls.query.get(model_id)
+   
+    if not model:
+        abort(make_response({"message":f"{cls.__name__} {model_id} not found"}, 404))
+    
+    return model
+
```

Comment:
I love the helper function.  It would be good to have one that validates the request body for post/patch requests.